internal/cmd/eventgen/linkslice.go

```diff
@@ -18,49 +18,37 @@ package main
 
 import (
 	_ "embed"
-	"text/template"
+	"fmt"
 
+	"github.com/eiffel-community/eiffelevents-sdk-go"
```

Comment:
Is this no longer an "internal dependency" and should be on its own like in the previous iteration?

Reply:
Nope, that's an oversight. Correcting commit pushed.